cmd/healthchecker/options/options_windows.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
```

Comment:
The options file is nearly identical. The only difference is SystemdService vs Service. I'd rename the `options.SystemdService` field to `options.ServiceName`.

I think using runtime.GOOS == "windows" for the flag name and description difference is reasonable here considering only 1 of 100 lines need to be OS specific.